Library/Formula/poppler.rb

```diff
@@ -51,8 +53,12 @@ def install
 
     if build.with? "qt"
       args << "--enable-poppler-qt4"
+    elsif build.with? "qt5"
+      ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["qt5"].opt_prefix}/lib/pkgconfig"
```

Comment:
Strange. Should be done by [this block](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/extend/ENV/super.rb#L134-L138).